xclim/sdba/__init__.py

```diff
@@ -63,6 +63,11 @@
 else:
     del polyval
 
+from . import adjustment
```

Comment:
What about putting the adjustment methods right into the sdba namespace ?

Reply:
right, makes more sense.